docs/usage/diff.rst

```diff
@@ -34,3 +34,9 @@ Examples
        +135 B    -252 B file2
     added           0 B file4
     removed         0 B file3
+
+    $ borg diff testrepo::archive1 archive3 --json-lines
+    {"changes": [ {"type": "mode", "oldmode": "-rw-r--r--", "newmode": "-rwxr-xr-x"} ], "path": "file1"}
+    {"changes": [ {"type": "modified", "added": 135, "removed": 252} ], "path": "file2"}
+    {"changes": [ {"type": "added", "size": 0} ], "path": "file4"}
+    {"changes": [ {"type": "removed", "size":0} ], "path": "file3"}
```

Comment:
also a style thing, but maybe more natural for the reader: put the path entry to the left, followed by the changes entry?

Reply:
ok, originally just coded like the text output, which had changes, then file name (to make parsing possible), but I agree, I also prefer path first.